config-templates/azure/azuredisk-csi-driver/1.30.3/metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "1.30.3",
+    "revision": "1",
+    "enabled": "true",
+    "status": "supported",
+    "description": "Azure Disk CSI Driver",
+    "tags": [
+        {
+            "name": "provider",
+            "value": "Azure"
+        },
+        {
+            "name": "offering-name",
+            "value": "azuredisk-csi-driver"
+        },
+        {
+            "name": "offering-type",
+            "value": "non-sds"
+        },
+        {
+            "name": "provider-url",
+            "value": "https://portal.azure.com"
+        },
+        {
+            "name": "support-msg",
+            "value": "For support contact Microsoft Azure"
+        },
+        {
+            "name": "provider-logo",
+            "value": "logo/azuredisk-csi-driver.svg"
+        },
+        {
+            "name": "supported-envs",
+            "value": "Azure"
+        },
+        {
+          "name": "supported-ocp-versions",
```

Comment:
Please u[date the OCP versions. Right now supported OCP versions in IBM cloud are
4.12, 4.13, 4.14, 4.15, 4.16. You can keep from 4.13 to 4.16. I hope this was tested on 4.16. IF not we will quickly perform one test on 4.16

Reply:
Successfully tested on 4.16 cluster: https://github.ibm.com/alchemy-containers/armada-storage/issues/6685